src/mgr/Action.cc

```diff
@@ -67,7 +67,7 @@ Mgr::Action::add(const Action &)
 void
 Mgr::Action::respond(const Request &request)
 {
-    debugs(16, 5, HERE);
+    debugs(16, 5, "");
```

Comment:
This does not look like a constructor to me, but I agree that this code can be improved. This PR automatically removes HERE, and nothing else should be required.